### ISSUE TITLE
No crafting recipes

### ISSUE THREAD

**Issue by KamiSami** [ejektaflex/NaturalPledge#7]
https://imgur.com/fGcAAW7
https://imgur.com/Oi8QAuQ

Is this bug possible to confirm if it happens to you aswell? Or do you need some logs for this? This happens on 3.0.2 version of the mod

**Comment by ejektaflex** [comment]
I might need logs. Botania/NaturalPledge/JEI combo is showing recipes in JEI. I've got to check the book again to make sure those are showing.

**Comment by KamiSami** [comment]
Pastebin didn't allow me to upload it in there as size of log was too large, i uploaded the text file in uploadfiles.io if that's fine https://ufile.io/f35vq

**Comment by ejektaflex** [comment]
That's... a big log. In the future, try gist.github.com if you can :)

Did you open the lexicon and flip to that page while the game/log was running? It doesn't seem to be saying much right now.

**Comment by KamiSami** [comment]
I don't think i did, not sure what happened but next time i logged in i gained chaining intention and the new millenium and glass of time achievements just by logging in to my server and now every crafting recipe is visible.. I don't know what i did different this time other than restarted server, but at least the recipes and everything is now visible, should this be closed now? Oh also a questions, what does it need to do ritual of gaia 3? I built the layout correctly but the book does not say what do i offer to it, i tried gaia ingot but the fight seemed same as before

**Comment by ejektaflex** [comment]
It should be a gaia spirit ingot. I'll have to look into it - Vazkii changed it so that the Gaia Guardian no longer has a public field telling us whether or not it's Gaia I or Gaia II, so the conditions for us to change it to Gaia III have changed.

Are you sure that it's not any different? From what I can tell, a message should be sent in chat if wearing an emblem, killing it should awaken your emblem, and there may be some special drop(s).

**Comment by KamiSami** [comment]
Yeah, it does not seem any different, im wondering, can it be possible the mod is conflicting with another mod? Like ExtraBotany as i have that aswell
Edits: Nevermind about ExtraBotany, it didn't work even without that off from server

**Comment by ejektaflex** [comment]
I've got to look into the way that I'm checking whether it's Gaia II. It is probably stemming from there.

**Comment by SSKirillSS** [comment]
At the moment, the bug with Gaia III is fixed? I still can't summon her.
